galasa-extensions-parent/dev.galasa.ras.couchdb/src/main/java/dev/galasa/ras/couchdb/internal/CouchdbDirectoryService.java

```diff
@@ -48,21 +48,22 @@
 import dev.galasa.framework.spi.ras.RasTestClass;
 import dev.galasa.framework.spi.ras.ResultArchiveStoreFileStore;
 import dev.galasa.extensions.common.api.LogFactory;
+import dev.galasa.extensions.common.couchdb.pojos.Row;
+import dev.galasa.extensions.common.couchdb.pojos.ViewResponse;
+import dev.galasa.extensions.common.couchdb.pojos.ViewRow;
+import dev.galasa.extensions.common.api.HttpRequestFactory;
 import dev.galasa.ras.couchdb.internal.pojos.Find;
 import dev.galasa.ras.couchdb.internal.pojos.FoundRuns;
 import dev.galasa.ras.couchdb.internal.pojos.IdRev;
-import dev.galasa.ras.couchdb.internal.pojos.Row;
 import dev.galasa.ras.couchdb.internal.pojos.TestStructureCouchdb;
-import dev.galasa.ras.couchdb.internal.pojos.ViewResponse;
-import dev.galasa.ras.couchdb.internal.pojos.ViewRow;
 
 public class CouchdbDirectoryService implements IResultArchiveStoreDirectoryService {
 
-    private final Log             logger ;
-    private final LogFactory      logFactory ;
-    private final HttpRequestFactory           requestFactory;
+    private final Log logger;
+    private final LogFactory logFactory;
+    private final HttpRequestFactory requestFactory;
 
-    private static final Charset  UTF8   = Charset.forName("utf-8");
+    private static final Charset UTF8 = Charset.forName("utf-8");
```

Comment:
Can this be done without a .forName ?

Reply:
Done using the java.nio `StandardCharsets.UTF_8` constant